src/test/run-pass/align-struct.rs

```diff
@@ -193,4 +200,14 @@ pub fn main() {
     assert_eq!(mem::align_of_val(&a.b), 1);
     assert_eq!(mem::size_of_val(&a), 16);
     assert!(is_aligned_to(&a, 16));
+
+    let mut arr = [0; 0x10000];
+    arr[0] = 132;
+    let large = AlignLarge {
+        stuff: arr,
+    };
```

Comment:
Good point, I thought an upstream change might result in that but I didn't think to avoid it with `box`.